buildSrc/build.gradle.kts

```diff
@@ -28,4 +29,8 @@ dependencies {
   // Could not create an instance of type aQute.bnd.gradle.BundleTaskExtension.
   // Cannot change attributes of configuration ':native-image-tests:compileClasspath' after it has been locked for mutation
   implementation("biz.aQute.bnd:biz.aQute.bnd.gradle:7.1.0")
+
+  implementation("org.jetbrains.kotlin:kotlin-gradle-plugin-api:2.1.21")
```

Comment:
Ideally I'd use toml for this, but it's not so simple.

Reply:
https://github.com/cashapp/redwood/blob/9571c679d736ee36e587773abe7a811936254025/build-support/settings.gradle#L6-L12